cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelperTest.java

```diff
@@ -206,6 +215,81 @@ public void testAddingThrottlesWithPreExistingThrottles() throws InterruptedExce
     assertExpectedThrottledReplicas(kafkaZkClient, TOPIC1, "1:1");
   }
 
+  @Test
+  public void testDoNotModifyExistingWildcardReplicaThrottles() {
```

Comment:
This test and `testDoNotRemoveThrottlesForInProgressTasks` seems to have some duplicate parts. I am curious if we can extract those common code to a function?

Reply:
It's totally possible to extract common code to a function. However, IMO, that would decrease the readability of both test cases. I would suggest considering refactoring out the common part when we have more unit tests testing the same workflow since currently there are only 2 such unit tests (not too much redundancy) and each of them is readable and easy to follow the test workflow.